packages/app/src/components/Common/ImageCropModal.tsx

```diff
@@ -93,10 +95,22 @@ const ImageCropModal: FC<Props> = (props: Props) => {
     }
   };
 
+  // Convert base64 Image to blob
+  const convertBase64ToBlob = async(base64Image: string) => {
+    const parts = base64Image.split(';base64,');
+    const imageType = parts[0].split(':')[1];
+    const decodedData = window.atob(parts[1]);
+    const uInt8Array = new Uint8Array(decodedData.length);
+    for (let i = 0; i < decodedData.length; ++i) {
```

Comment:
it doesn't change much but let's make it to `i++`

I found the article that shows the easier way to convert base64 string to blob
see: https://ionic.io/blog/converting-a-base64-string-to-a-blob-in-javascript